Makefile

```diff
@@ -42,9 +43,13 @@ else
 	REGISTRY = $(RP_IMAGE_ACR)
 endif
 
-RP_IMAGE_LOCAL ?= aro
-ARO_PORTAL_BUILD_IMAGE ?= $(RP_IMAGE_LOCAL)-portal-build
-ARO_BUILDER_IMAGE ?= $(RP_IMAGE_LOCAL)-builder
+# local images
+LOCAL_ARO_RP_IMAGE ?= aro
+LOCAL_ARO_PORTAL_BUILD_IMAGE ?= $(LOCAL_ARO_RP_IMAGE)-portal-build
+LOCAL_ARO_RP_BUILD_IMAGE ?= $(LOCAL_ARO_RP_IMAGE)-build
+LOCAL_AZ_EXT_ARO_IMAGE ?= azext-aro
```

Comment:
@shubhadapaithankar I think this has implications on CI, so we will need to resolve what envvars we use and what order these PRs merge in.